tests/unit/lib/build_module/test_build_strategy.py

```diff
@@ -833,7 +839,7 @@ def test_wrapper_with_or_without_container(self, mocked_read, mocked_write, runt
             use_container,
         )
 
-        build_definition = FunctionBuildDefinition(runtime, "codeuri", "packate_type", X86_64, {}, "handler")
+        build_definition = FunctionBuildDefinition(runtime, "codeuri", None, "packate_type", X86_64, {}, "handler")
```

Comment:
I think this is an old typo, but can we change it to package_type from packate_type for the occurrences?